services/patient-service/patient-service.js

```diff
@@ -52,7 +52,18 @@ class PatientService {
    * @returns {Array<PatientObject>} Patients matched to the nurse ID
    */
   async returnNursePatients(idNurse) {
+    if (!idNurse) {
+      throw new BadRequestError("Please provide an Employee id");
+    }
+
+    const employee = await EmployeeModel.findById(idNurse);
+
+    if (!employee) {
+      throw new NotFoundError("Employee was not found");
+    }
```

Comment:
We can be more specific. The function is to return the patients of a nurse, so change `employee` word to `nurse`

```suggestion
    if (!idNurse) {
      throw new BadRequestError("Please provide a Nurse id");
    }

    const nurse = await EmployeeModel.findById(idNurse);

    if (!nurse) {
      throw new NotFoundError("Nurse was not found");
    }
```